src/main/webapp/common_util.js

```diff
@@ -44,3 +44,16 @@ function clearElement(elementId) {
     elem.removeChild(elem.lastChild);
   }
 }
+
+/**
+ * @return API Key for YouTube Data API
+ */
+function getApiKey() {
+  $(function() {
+    $.get('apiKey', function(data) {
+      return data;
```

Comment:
jslint error

Reply:
This JSLint error is errant and due to how the function is structured.